modules/ROOT/pages/hardware-and-software-requirements.adoc

```diff
@@ -43,7 +43,7 @@ Mule runs on any operating systems that runs a supported Java Runtime Environmen
 [NOTE]
 Mule might work with other software and operating systems, but those have not been tested. MuleSoft recommends that you run Mule on tested software and operating systems versions.
 
-This version of Mule Runtime is bundled with the Runtime Manager agent plugin version 2.1.9.
+This version of Mule Runtime is bundled with the Runtime Manager agent plugin version 2.3.0. For Runtime Manager Agent compatibility, see xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes].
```

Comment:
version came from info at bottom of https://docs.mulesoft.com/release-notes/mule-runtime/mule-4.2.0-release-notes#initial-4-2-0-release